### ISSUE TITLE
vpp-fruc is not workling correctly

### ISSUE THREAD

**Issue by rainman74** [rigaya/NVEnc#565]
Tests:
--vpp-fruc double -> works and is incredibly fast ✅🚀
--vpp-fruc fps=60000/1001 -> is ignored, double is used ❌
--vpp-fruc fps=50000/1001 -> is ignored, double is used ❌
--vpp-fruc fps=50.0 -> works, but you get debug messages ❌
--vpp-fruc fps=60.0 -> works, but you get debug messages ❌

You get tons of these debug messages:
```
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4351 kb/s, remain 0:00:19, GPU 20%, VE 6%, est out size 9.2MB
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4357 kb/s, remain 0:00:17, GPU 19%, VE 6%, est out size 9.1MB
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4357 kb/s, remain 0:00:17, GPU 19%, VE 6%, est out size 9.2MB
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4359 kb/s, remain 0:00:17, GPU 15%, VE 5%, est out size 9.3MB
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4367 kb/s, remain 0:00:18, GPU 21%, VE 5%, est out size 9.4MB
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4374 kb/s, remain 0:00:15, GPU 26%, VE 6%, est out size 9.3MB
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4
Optical Flow Grid Size: 4 fps, 4380 kb/s, remain 0:00:15, GPU 14%, VE 4%, est out size 9.4MB
```

**Comment by rainman74** [comment]
and in help output there is missing a carriage return after ...(fast):

`   --vpp-fruc [<param1>=<value>][,<param2>=<value>][...]
     enable frame rate up conversion filter.
    params
      double                     double frame rate **(fast)**      fps=<int>/<int> or <float> target frame rate
`

**Comment by rigaya** [comment]
Thank you for your report, fps=x/x will be fixed in next release (you can still try --vpp-fruc fps=59.94 at this time for 60000/1001).

> You get tons of these debug messages:

Yes, this is unavoidable as this is coming from FRUC library (or NVOF library) and I were not able to find ways to suppress them.

Please note that ```fps``` mode ( = non ```double``` mode) is really slow as it requires re-initialization every frame, due to limitation of the library.

**Comment by rigaya** [comment]
The fps in fraction issue will be fixed in [NVEnc 7.44](https://github.com/rigaya/NVEnc/releases/tag/7.44).

**Comment by rainman74** [comment]
Problem is fixed!
